README.md

```diff
@@ -1,2 +1,151 @@
+    - [Live Demo](#Demo)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Sema] <a name="about-project"></a>
+
+**[Sema]** is a fullstack application built with ruby on rails and postgres to mimic a social commentary platform.
+## 🛠 Built With <a name="built-with"></a>
+- Ruby
+- PostgreSQL
+- Ruby on Rails
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- [ ] **[like comments]**
+- [ ] **[comments]**
+- [ ] **[Posts]**
+- [ ] **[Userprofile]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+- Laptop
+- Github
+- Git
+- Ruby
+- Linters 
+- Ruby on Rails
+
+### Prerequisites
+
+In order to run this project you need:
+- To install Ruby following these steps:
+- Install the latest version on ruby+devkit [rubyinstaller](https://rubyinstaller.org/downloads/)
+- Follow the installation steps [here](https://stackify.com/install-ruby-on-windows-everything-you-need-to-get-going/)
+- install the [linters for Ruby](https://github.com/microverseinc/linters-config/tree/master/ruby)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Samuelwanza/sema.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd sema
+  rails s
+```
+<!--DEMO--->
+### Demo
+[]()
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Samuel Wanza**
+
+- GitHub: [@githubhandle](https://github.com/Samuelwanza)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/samuel-munguti/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Userdashboard]**
+- [ ] **[Management dashboard]**
+- [ ] **[Most viewed comments and post]**
+- [ ] **[personalise recommendations]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Samuelwanza/sema/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project give us a ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse Team.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT]() licensed.
```

Comment:
- [ ] Please ensure that clicking the MIT link redirects to the created license file.:pray: